src/config/constants.ts

```diff
@@ -103,6 +105,7 @@ export const HOST_STATUS_DETAILS: { [key in Host['status']]: string } = {
   resetting: 'This host is resetting the installation.',
   'resetting-pending-user-action':
     'Host already booted from disk during previous installation. To finish resetting the installation please boot the host into Discovery ISO.',
+  'added-to-existing-cluster': 'Host is added to existing cluster',
```

Comment:
```suggestion
  'added-to-existing-cluster': 'This host is being added to existing cluster.',
```

- consistent with other status messages
- quoting Yevgeny: "it means that we have installed RCHOS with the ignition on the node and rebooted it. Once node is rebooted we have no way of monitoring its progress ( kubelet connection to OCP cluster etc'). So from our point of view we have finished with the node, but user must see a meessage that he must no monitor its progress till it is actually added to the OCP cluster"

IIUC we inform the user that the host is still in process of adding it to the cluster.